cmd/scheduler/app/app.go

```diff
@@ -109,7 +110,6 @@ func Run() {
 				EtcdSpaceQuota:          opts.EtcdSpaceQuota,
 				EtcdCompactionMode:      opts.EtcdCompactionMode,
 				EtcdCompactionRetention: opts.EtcdCompactionRetention,
-				EtcdClientHTTPPorts:     opts.EtcdClientHTTPPorts,
```

Comment:
Do we need to set it? If removed, use default one?

Reply:
We only need a gRPC client port for cron.